Setup/Product.wxs

```diff
@@ -63,13 +63,13 @@
         <File Source="..\GitExtensions\bin\$(var.Configuration)\GitExtensions.exe.config" />
       </Component>
       <Component Id="System.IO.Abstractions.dll" Guid="*">
-        <File Source="..\GitUI\bin\$(var.Configuration)\System.IO.Abstractions.dll" />
+        <File Source="..\gitextensions\bin\$(var.Configuration)\System.IO.Abstractions.dll" />
```

Comment:
❕ Casing should be consistent

Reply:
Please hold off on the formatting review. Look. I do it like this on purpose. I will add a formatting commit once functionality is taken care of.

Reply:
<kbd>Ctrl</kbd> + <kbd>K</kbd> + <kbd>D</kbd> does wonders.

Reply:
@vbjay This isn't about formatting. It's a comment that some of the document uses `GitExtensions` as a folder name, and other parts of the document uses `gitextensions` for the same folder.

Reply:
Phone scrolled.  It was so helpful.  :-1:  I'll adjust the script.  I am using linqpad to analyze xml and files in release folder and it used the actual case of the path.  Kinda helps me find what was modified in the editor.  I have no problem fixing the case but at the moment my priority is does it include the files needed?  I still have some tweaking to do.  I just wanted to show where I was.